### ISSUE TITLE
lighthouse failure:  Background and foreground colors do not have a sufficient contrast ratio.

### ISSUE THREAD

**Issue by daniel-packard** [modernweb-dev/web#1588]
The text on https://modern-web.dev/ struck me as hard to read - and according to lighthouse does not meet accessibility guidelines for contrast:

<img width="469" alt="Screen Shot 2021-07-27 at 7 10 02 PM" src="https://user-images.githubusercontent.com/1051245/127243349-d05fb791-fc55-4f4f-87c8-be6efa2154ae.png">

Would you welcome a pull request to adjust the contrast between the foreground text and the background color?

**Comment by daniel-packard** [comment]
Looks as if someone fixed it by making the text white - bravo and thank you to whoever did this

<img width="617" alt="Screen Shot 2021-08-14 at 12 04 22 PM" src="https://user-images.githubusercontent.com/1051245/129454870-9fff8600-3198-40ba-ba8c-1f6c6ddc957c.png">

closing as already fixed
